### ISSUE TITLE
Selective Paste suggestion

### ISSUE THREAD

**Issue by EugeneMichel** [rsjaffe/MIDI2LR#68]
Wanted to say thanks for all the great work so far on MIDI2LR! So much has been improved since 0.3!

I just tried the new 0.7 version and selective paste settings dont seem to work for me. It pastes some parameters but no matter what settings I select in Set Options menu its always the same. I do set the setting to PasteSelectedSettings

Since some images need some settings and some images need all settings pasted, I want to suggest maybe an easier way to implement it - When "Copy Settings" button is pressed, it would act the same way as pressing the "Copy..." button on the lower left corner in the Develop module. When you press Copy a dialog opens and allows you to choose what to copy. So instead of clicking the button with the mouse, "Copy Settings" in MIDI2LR would open that dialog, user would select needed settings and then "Paste Settings" would paste whatever was selected in the Copy Settings dialog in LR. 

I use copy settings often and it would take time to go into Set Options each time a different setting needs to be copied, this way would be faster. Another program that I used before acted this way.

Just a suggestion and only if its an easy fix. Thanks for your work!

**Comment by rsjaffe** [comment]
The paste all settings is still available. I've assigned one button to paste all settings and another to paste selected settings. I checked some of the paste selected settings check boxes, and was able to change which settings were pasted.

However, I understand your suggestion about popping up a selection dialog for pasting settings. I could work on putting that as yet another option for those who want a popup for each paste--it could retain the last used set of options. Shouldn't be too difficult to do, as it just requires another copy of the dialog tab I've currently put in place, along with a table to track its state.
